core-java-modules/core-java-collections-list-3/src/main/java/com/baeldung/list/difference/FindDifferencesBetweenLists.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.list.difference;
+
+import com.google.common.collect.Sets;
+import org.apache.commons.collections4.CollectionUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FindDifferencesBetweenLists {
+
+    public static <T> List<T> differencesUsingPlainJavaCollections(List<T> listOne, List<T> listTwo) {
+        Set<T> differences = new HashSet<>(listOne);
+        differences.removeAll(listTwo);
+        return new ArrayList<>(differences);
```

Comment:
The example code also needs implementation and test for the text of the article:

```java
List<T> differences = new ArrayList<>(listOne); 
differences.removeAll(listTwo);
```